src/app/launcher/wizard.component.ts

```diff
@@ -15,6 +16,7 @@ import { WizardStep } from './wizard-step';
 @Component({
   encapsulation: ViewEncapsulation.None,
   selector: 'f8launcher-wizard',
+  changeDetection: ChangeDetectionStrategy.OnPush,
```

Comment:
Can you please explain why OnPush is used here? 

This means that the "value of the component can change only through user inputs and events". I'm concerned about being able to restore values after navigating away for GitHub auth -- none of the other wizard components are using this approach.

Reply:
@dlabrecq Sure, actually we get 'ERROR Error: ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value: 'false'. Current value: 'true'.' only in dev mode, so in order to address that I brought that in.

Since Angular2 by default runs change detection twice for dev mode.

If it's ok to have that error in dev mode, we can remove this or may be think of a better solution.

Reply:
The error is actually due to the scroll functionality. We still need to address that; however, when ChangeDetectionStrategy.OnPush is added, it blocks the scroller from making changes in the component. There is no longer an initial selection made when the user enters the wizard. That is, the step indicator has no initial selection.